addon/components/horizontal-bar-chart.js

```diff
@@ -237,13 +237,24 @@ const HorizontalBarChartComponent = ChartComponent.extend(FloatingTooltipMixin,
     };
   }),
 
+  /*
+   * Ensures that grouping labels will be on the left when all data is negative or 0
+   * @private
+   * @param {Object}
+   * @return {Boolean}
+   */
+  _isNotPositive: function(d) {
+    return ((d.value < 0) || ((d.value === 0) && this.get('hasAllNegativeValues')));
```

Comment:
I feel like there are some unnecessary parens here. I think the outer-most parens can be removed